common/lib/protocol-definitions/package.json

```diff
@@ -211,6 +212,14 @@
         "EnumDeclaration_FileMode": {
           "forwardCompat": false
         }
+      },
+      "0.1027.1000":{
+        "EnumDeclaration_FileMode": {"forwardCompat": false},
```

Comment:
@NicholasCouri do these breaks look like what you would expect? not sure if they are all yours or not.

Reply:
yes, I believe they are.

Reply:
no rules. sometimes the file gets reserialized which adds the new lines.